charts/metering-ansible-operator/templates/crds/meteringconfig.crd.yaml

```diff
@@ -367,55 +367,147 @@ spec:
                     authProxy:
                       type: object
                       properties:
+                        enabled:
+                          type: boolean
                         authenticatedEmails:
                           type: object
                           properties:
+                            enabled:
+                              type: boolean
                             createSecret:
                               type: boolean
                             data:
                               type: string
-                            enabled:
-                              type: boolean
+                              minLength: 1
                             secretName:
                               type: string
+                              minLength: 1
+                          oneOf:
```

Comment:
This would allow you to 
```
reporting-operator:
  spec:
    authProxy:
      authenticatedEmails:
        enabled: false
```
or:
```
reporting-operator:
  spec:
    authProxy:
      authenticatedEmails:
        enabled: true
        createSecret: false
        secretName: existing-secret-name
```
or: 
```
reporting-operator:
  spec:
    authProxy:
      authenticatedEmails:
        enabled: true
        createSecret: true
        data: non-empty-data
        # secretName is optional
        secretName: new-secret-name
```

Reply:
Note: `authProxy.authenticatedEmails` is not part of the `_tls_overrides` variables.